app/models/drug_safety_update.rb

```diff
@@ -15,20 +15,33 @@ def publish!
 
     begin
       update_type = self.update_type || 'major'
+
+      return false if not_published? && !save_first_published_at
+
       publish_request = publishing_api.publish(content_id, update_type)
       rummager_request = rummager.add_document(
         search_document_type,
         base_path,
         indexable_document.to_json,
       )
 
-      publish_request.code == 200 && rummager_request.code == 200
-
+      publish_request.code == 200 && rummager_request.code == 202
     rescue GdsApi::HTTPErrorResponse => e
       Airbrake.notify(e)
+
+      false
     end
   end
 
+  def save_first_published_at
+    self.first_published_at = Time.zone.now
+    presented_document = DocumentPresenter.new(self).to_json
+
+    put_request = publishing_api.put_content(self.content_id, presented_document)
+
+    put_request.code == 200
+  end
```

Comment:
Based on the discussions happening on #732 we don't need the final line here checking the status code.  Either the call succeeds or it raises an exception (which will be handled by the `rescue` in `publish!` above).  With that in mind the guard clause in `publish!` could be rewritten as:

```ruby
save_first_published_at false if not_published?
```

Because if it fails the exception will make the whole `publish!` method fail.  Might also want to make `save_first_published_at` a private method.